rs/registry/canister/src/registry.rs

```diff
@@ -1125,8 +1150,44 @@ mod tests {
         let req = HighCapacityRegistryAtomicMutateRequest {
             mutations,
             preconditions: vec![],
-            // Since the `too_large_value` is built with serializing maximum timestamp length to 10 bytes
-            // the tipping point of `+ 1` will be there only if we account the timestamp of full serialized 10 bytes.
+            // Since the `too_large_value` is built with serializing maximum
+            // timestamp length to 10 bytes the tipping point of `+ 1` will be
+            // there only if we account the timestamp of full serialized 10
+            // bytes.
+            timestamp_nanoseconds: u64::MAX,
+        };
+
+        registry.changelog_insert(1, req);
+    }
+
+    #[test]
+    #[should_panic(expected = "[Registry] Transaction rejected because delta would be too large")]
+    fn test_changelog_insert_delta_too_large_but_no_prime_mutation_large() {
+        let mut registry = Registry::new();
+
+        // This is just (slightly) more elaborate+realistic version of the data
+        // in the previous test, but we are essentially doing the same thing:
```

Comment:
Let's refer to the name of the test, as the order of tests in this file isn't guaranteed to stay the same forever.